src/main/java/plansoft/skillsmanagement/level/LevelService.java

```diff
@@ -24,8 +24,21 @@
 	}
 
 	public void addLevel(Level level) {
+		List<Level> levels = new ArrayList<>();
+		levelRepository.findAll().forEach(levels::add);
+		for (int i = 0; i < levels.size(); i++) {
```

Comment:
Potresti usare gli iteratori quindi sfruttare il for each, in questo modo risulta più leggibile